technical-reports/format/terminology.md

```diff
@@ -100,3 +100,19 @@ The value of `$color-text-primary` is `#000000`, because `$color-text-primary` _
 ## Composite (Design) Token
 
 A design token whose value is made up of multiple, named child values. Composite tokens are useful for closely related style properties that are always applied together. For example, a typography style might be made up of a font name, font size, line height, and color.
+
+Here's [an example of a composite shadow token](https://design-tokens.github.io/community-group/format/#example-composite-token-example):
+```
```

Comment:
~~~suggestion

```json
~~~

Reply:
I'll go ahead and commit @kaelig's suggestion so that we get a clean build on this PR. (Hope you don'tmind @ChucKN0risK )